### ISSUE TITLE
should `ariaLabel` always be set to `title` when none has been provided?

### ISSUE THREAD

**Issue by meganrogge** [microsoft/vscode#236247]
To prevent a button from missing an `ariaLabel` example https://github.com/microsoft/vscode/issues/236094, should we always fallback to the `title` here?

cc @isidorn 

https://github.com/microsoft/vscode/blob/e3d033cf31ffba88c2185b1f79f6e57caea39fa3/src/vs/base/browser/ui/button/button.ts#L263-L268

**Comment by jrieken** [comment]
Unassigning because I don't own the button component. Unsure if this is @sbatten or @bpasero

**Comment by meganrogge** [comment]
Verify via code review

**Comment by roblourens** [comment]
It's confusing that this is done from the label/icon setters instead of in the Button constructor, but seems fine
